net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
```

Comment:
@fbrosson However, my patchset relies on the autoconf stuff:
```diff
diff --git a/configure.in b/configure.in
index d8c6668..fd6b42d 100644
--- a/configure.in
+++ b/configure.in
@@ -9,6 +9,7 @@ test ".$CFLAGS" = "." && CFLAGS=" "
```
Perhaps we still need to download from GitHub?

Reply:
Oops, you are probably right. I had not noticed the tarball on SourceForge did not include the configure.in file.
So if you would like to revert to the tarball from GitGub, that's OK for me.
@alaviss, what do you think?

BTW, @sfanxiang, don't forget to try @alaviss's suggestion to use the LIBS variable:
```
	LIBS="-Wl,--as-needed -lbnetapi -lnetwork" runConfigure ./configure
```
It may allow you to drop parts of the patchset.
(A smaller patchset means a better recipe :)
Thanks!